features/root.yml

```
@@ -0,0 +1,6 @@
+name: :root
+description: The `:root` pseudo-class matches the root element of the document.
```

To make this a little more descriptive:

```suggestion
description: The `:root` pseudo-class matches the root element of the document, usually the `<html>` element.
```